experimental/ExtendTeamsforM365/client/pages/categoryDetail.html

```diff
@@ -17,7 +28,7 @@
     <div id="content">
         <p>Loading...</p>
     </div>
-    <table id="products">
+    <table id="products" style="overflow: auto;">
```

Comment:
Move this to css file.

Reply:
Changed and shifted to northwind-teams.css